autolift-scalaz/src/main/scala/autolift/scalaz/Scalaz.scala

```diff
@@ -2,10 +2,22 @@ package autolift
 
 import autolift.scalaz._
 
-object Scalaz extends Syntax with Context with Reexports with Implicits{
-	implicit def mkM[Obj, Fn](implicit lift: ScalazLiftMap[Obj, Fn]): ScalazLiftMap.Aux[Obj, Fn, lift.Out] = lift
+object Scalaz extends Syntax with Context with Reexports with Implicits with LiftMapSyntax with ScalazLiftBindSyntax {
+
+  protected type Functor[T[_]] = _root_.scalaz.Functor[T]
+  protected type Monad[T[_]] = _root_.scalaz.Bind[T]
```

Comment:
This is bad. Should not redefine a `Monad` type with `Bind`. Ok, I see you kept the implementation in line but the semantics are a bit confusing.